src/genie/libs/parser/iosxe/show_snmp.py

```diff
@@ -79,3 +79,139 @@ def cli(self, output=None):
 
         return ret_dict
 
+
+
+# ===================
+# Schema for:
+#  * 'show snmp user'
+# ===================
+class ShowSnmpUserSchema(MetaParser):
+    """Schema for show snmp user."""
+
+    schema = {
+        Any(): {
+            Optional("access-list"): str,
+            "auth-protocol": str,
+            "engine-id": str,
+            "group-name": str,
+            "priv-protocol": str,
+            "storage-type": str
+        }
+    }
+
+
+# ===================
+# Parser for:
+#  * 'show snmp user'
+# ===================
+class ShowSnmpUser(ShowSnmpUserSchema):
+    """Parser for show snmp user"""
+
+    cli_command = "show snmp user"
+
+    def cli(self, output=None):
+        if output is None:
+            output = self.device.execute(self.cli_command)
+
+        else: 
+            output = output
+
+        # User name: SNMPv3-alfa
+        # Engine ID: 800000090300002790FBCF00
+        # storage-type: nonvolatile	 active
+        # Authentication Protocol: SHA
+        # Privacy Protocol: AES128
+        # Group-name: ALFA
+        # 
```

Comment:
You don't need to copy the whole thing, I would just do one with all of the options. In this case, that is with and without whitelist.

Reply:
Once you do this, please put in a PR to Cisco's branch. After that, make sure to link the Cisco PR in the comments of this PR.